projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/DataJobCrudIT.java

```diff
@@ -68,7 +68,7 @@ public void testDataJobCrud() throws Exception {
       testDataJobPostCreateWebHooks();
 
       // Execute get swagger with no user
-      mockMvc.perform(get("/swagger-ui.html")
+      mockMvc.perform(get("/data-jobs/swagger-ui.html")
```

Comment:
Out of curiosity, should we think about a way to make this configurable in the future, in case the jobs' path changes again?

Reply:
A generic way would be with a configurable the spring/servlet context path, however this would require refactoring

Reply:
Gotcha